src/components/game/table/useContextMenuPreventer.js

```diff
@@ -0,0 +1,15 @@
+import { useEffect } from 'react';
+
+export const useContextMenuPreventer = () => {
+  useEffect(() => {
+    const preventContextMenu = e => {
+      e.preventDefault();
+      return false;
+    };
+
+    document.body.addEventListener('contextmenu', e => preventContextMenu(e));
+    return () => {
+      document.body.addEventListener('contextmenu', e => preventContextMenu(e));
```

Comment:
To musi być tu powtórzone na clean upie? Nie powinien być tu ten listener odklejany?

Reply:
to jest blad, mialo byc remove event listener xD

Reply:
good catch